wordclock_config/wordclock_config.example.cfg

```diff
@@ -89,17 +94,30 @@ activate = True
 
 [plugin_time_default]
 activate = True
-# animate new time with typewriter animation
-typewriter = false
-typewriter_speed = 10
+# animate new time. Options: no_animation, typewriter, fadeOutIn
```

Comment:
Nice! Implementation wasn't properly updated to a (hopefully growing) number of animations so far.